istio/pkg/controllers/servicecenter/client/client.go

```diff
@@ -0,0 +1,180 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package client
+
+import (
+	"sync"
+	"time"
+
+	"github.com/apache/servicecomb-service-center/istio/pkg/event"
+
+	"github.com/go-chassis/cari/discovery"
+	"github.com/go-chassis/sc-client"
+	"istio.io/pkg/log"
+)
+
+const (
+	// Time in seconds to wait before re-registering a deleted Servicecomb Service Center Watcher service
+	REREGISTER_INTERVAL time.Duration = time.Second * 5
+)
+
+// Servicecomb Service Center go-chassis client
+type Client struct {
+	client                  *sc.Client
+	cacheMutex              sync.Mutex
```

Comment:
from the response i can see, u should not name it as Client, and the comments is also misleading the concept.